slave-dotnet10/test/run

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+#
+# Test the Jenkins image.
+#
+# IMAGE_NAME specifies the name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+# make sure oc is installed
+docker run ${IMAGE_NAME} oc
+# make sure npm and nodejs are available
```

Comment:
Our builder container also has npm installed for frontend dependencies. I wonder if we need that in the jenkins slave. Either way, the comment should probably get updated. i.e. "make sure dotnet" is available.